src/ansi.rs

```diff
@@ -1551,6 +1554,28 @@ where
 
                 handler.clear_line(mode);
             },
+            ('k', [b' ']) => {
+                // SCP control
+                //
+                // Changes from ECMA-48 as proposed by the BiDi draft proposal:
+                //  * Recognize value 0 for Ps1
```

Comment:
What purpose does this have for applications? Having a right-to-left "default" seems unfeasible to me. So effectively this would just be another way to use `1`.